spec/labor/rate_limit_checker_spec.rb

```diff
@@ -4,65 +4,59 @@
   let(:user) { create(:user) }
   let(:article) { create(:article, user_id: user.id) }
 
-  describe "self.daily_account_follow_limit " do
-    it "returns the value set in SiteConfig.rate_limit_follow_count_daily" do
-      expect(described_class.daily_account_follow_limit).to eq(SiteConfig.rate_limit_follow_count_daily)
-    end
-  end
-
   describe "#limit_by_action" do
+    let(:rate_limit_checker) { described_class.new(user) }
+
     it "returns false for invalid action" do
-      expect(described_class.new(user).limit_by_action("random-nothing")).to eq(false)
+      expect(rate_limit_checker.limit_by_action("random-nothing")).to eq(false)
     end
 
     it "returns true if too many comments at once" do
-      create_list(:comment, 10, user_id: user.id, commentable_id: article.id)
-      expect(described_class.new(user).limit_by_action("comment_creation")).to eq(true)
+      count = SiteConfig.rate_limit_comment_creation + 1
+      create_list(:comment, count, user_id: user.id, commentable_id: article.id)
```

Comment:
I haven't tested but would it be possible to mock or lower the rate limit as a precondition? Creating 10 comments in the DB when we can create just 2 or 3 it's a waste :D

Reply:
Good call, reduced the number of objects created.